src/DayCounts.jl

```diff
@@ -113,35 +113,36 @@ This ensures that:
  - [EMU and Market Conventions: Recent Developments, ISDA - BS:9951.1](https://www.isda.org/a/AIJEE/1998-ISDA-memo-EMU-and-Market-Conventions-Recent-Developments.pdf), §4. The Actual/Actual Day Count Convention.
 """
 struct ActualActualICMA <: DayCount
-    schedule::StepRange{Date,Month}
+    frequency
```

Comment:
Could give this a type: I guess `Float64`, unless it is always an integer?

Reply:
Frequency is the number of coupons per year and that should always be integer. An enum would also be fine I think, e.g. `@num Frequency Monthly=12, Bimonthly=6, Quarterly=4, Semiannual=2, Annual=1`.

Any preference? `Int` or `Enum`?